sql/src/test/java/org/apache/druid/sql/avatica/DruidStatementTest.java

```diff
@@ -113,8 +114,13 @@ public void tearDown() throws Exception
   public void testSignature()
   {
     final String sql = "SELECT * FROM druid.foo";
-    final DruidStatement statement = new DruidStatement("", 0, null, sqlLifecycleFactory.factorize(), () -> {
-    }).prepare(sql, -1, AllowAllAuthenticator.ALLOW_ALL_RESULT);
+    final DruidStatement statement = new DruidStatement(
+        "",
+        0,
+        new QueryContext(),
+        sqlLifecycleFactory.factorize(),
+        () -> {}
+    ).prepare(sql, -1, AllowAllAuthenticator.ALLOW_ALL_RESULT);
```

Comment:
Nit: Maybe add a `createStatement(sql)` private util method for this to avoid code repetition.